pkg/xray/resource_xray_license_policy.go

```diff
@@ -20,143 +21,177 @@ func resourceXrayLicensePolicyV2() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Name of the policy (must be unique)",
```

Comment:
Is there any naming convention restriction here? `validation.StringIsNotEmpty`? Validation please

Reply:
No restrictions per Xray documentation. Validation added